utils/post_processing/restructure_borealis.py

```diff
@@ -0,0 +1,329 @@
+# Copyright 2019 SuperDARN Canada, University of Saskatchewan
+# Author: Liam Graham
+
+#
+# restructure_borealis.py
+# 2019-07-30
+# Command line tool for strucuturing Borealis files
+# for bfiq, antennas iq, and raw acf data into a smaller,
+# faster, and more usable format
+
+import deepdish as dd
+import numpy as np
+import sys
+
+
+def find_shared(data):
```

Comment:
I'm not sure about this. Seems like this could create a difference in data fields between files. For example, if num_sequences happened to be the same in all records of a file by coincidence, would that become a shared field? I'd rather have the way that field names are formatted be defined in this file. This file could then provide warnings on unknown fields, etc. Maybe better at the top of the file to define: bfiq_shared_fields = ['data_descriptors', ''.... ] and similar for other file types.

Reply:
Fields now exist in very long lists.